go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/speza/casbin-bolt-adapter
+module github.com/mtal3t/casbin-bolt-adapter
```

Comment:
Why commit this?

Reply:
rookie mistake :), was trying to fix `go get ..` on my fork..